core/serialize/serialize.cc

```diff
@@ -868,9 +868,12 @@ void SerializerImpl::unpickleFileTable(UnPickler &p, GlobalState &result) {
     // We don't count the empty file that we reserve for the invalid FileRef.
     files.reserve(1 + filesSize);
     files.initEmpty();
+    result.symbolsReferencedByFile.reserve(1 + filesSize);
+    result.symbolsReferencedByFile.emplace_back();
 
     for (int i = 0; i < filesSize; i++) {
         files.emplace(unpickleFile(p));
+        result.symbolsReferencedByFile.emplace_back();
```

Comment:
Rather than doing this in a loop, maybe we should use something like

```cpp
result.symbolsReferencedByFile = vector<UnorderedSet<SymbolRef>>{1 + filesSize, {}};
```